src/domElements.js

```diff
@@ -0,0 +1,138 @@
+// Thanks to ReactDOMFactories for this handy list!
+
+export default [
```

Comment:
how do you make sure feature detection based results are always compatible with the list when using SSR?